### ISSUE TITLE
Feature request: option to permanently acknowledge the iPython console "mute inline plotting" warning

### ISSUE THREAD

**Issue by nukamoi** [spyder-ide/spyder#21279]
Each time a new iPython console is opened and plot is created, the console shows the warning: 

> IMPORTANT - Figures are displayed in the Plots pane by default. To make them also appear inline in the console, you need to uncheck "Mute inline plotting" under the options menu of Plots.

and after a while, this warning becomes a small nuisance.

I would be very grateful for an option to permanently acknowledge this warning, to prevent it from showing the next time I open a new console.

Suggestions would be:
-  a tick-box in Preferences > iPython console > Graphics (or similar location)
-  a tick box in the "hamburger" menu of the Plots pane

Thanks!

![image](https://github.com/spyder-ide/spyder/assets/141940373/1693f50d-a58f-4e98-8afe-55a0a17fa0aa)

**Comment by dalthviz** [comment]
Thank you for the feedback @nukamoi ! I'm not totally sure about the message logic but seems like currently the message is displayed always the first time an inline plot is generated per console. Thinking about it, it's true that when you already have seen the message a couple of times then seeing it again and again can feel a little bit like noise :thinking:

What do you think @spyder-ide/core-developers ?

**Comment by mrclary** [comment]
I agree with eliminating the warning from the IPython console and making it a one-time alert. I think we should consider using a QMessage box and only show it once when the inline backend or muting inline plotting preference changes, similar to Spyder Tour.

**Comment by ccordoba12** [comment]
> I agree with eliminating the warning from the IPython console and making it a one-time alert. I think we should consider using a QMessage box

There are some problems with this idea:

* Modal (i.e. QMessageBox) messages are quickly dismissed by users because they find them annoying.
* Plots and the IPython console are not visible at the same time in the default layout. So, I'm afraid if the message is not in the console and made quite visible, users will not spend time trying to find where the Plots pane is and they won't find their plots.

So, I propose two things to improve the situation:

* Add more text at the end of the message saying something like:

    > ([click here]() if you don't want to see this message anymore).

    Links are the only graphical elements we can add to the IPython console. If users click this one, then we'll change the value of an option in our config system that would disable the message for good the next time users create a plot in a new console. That would also have the advantage of being easily discoverable and not requiring to add an option to the interface.

* Add a link to the `Plots pane` part of the message text, so users can click on it and find where Plots is.

What do you think?

**Comment by mrclary** [comment]
> So, I propose two things to improve the situation:
> 
> * Add more text at the end of the message saying something like:
>   > (click here if you don't want to see this message anymore).
>   
>   
>   Links are the only graphical elements we can add to the IPython console. If users click this one, then we'd change the value of an option in our config system that would disable the message for good the next time users create a plot in a new console. That would also have the advantage of being easily discoverable and not requiring to add the option to the interface.

I like this idea.

> * Add a link to the `Plots pane` part of the message, so users can click on it and find where Plots is.

This is interesting. What if we could make the Plots Pane visible without taking focus. This would show the plots when a plot is made so that the user doesn't need to find it.

**Comment by Rational-IM** [comment]
I see from above that the "click here if you don't want to see this message anymore" was supposed to be implemented in v6.1.0. However, I'm using v6.1.0, and the option to deactivate this message is still not there. Any idea when this might be implemented? Obviously, not an urgent issue, but the message indeed becomes annoying for those of us who use Spyder daily.

**Comment by nukamoi** [comment]
While I don't see the message "(person) modified the milestone" in the issue's history, I do see that the current milestone is v6.2.0 (no due date). See the pane on the right hand side of this page.

**Comment by CAM-Gerlach** [comment]
FWIW, I was actually thinking the same thing, that we really should have an option to silence this warning permanently; it gets a bit annoying to see it every time once you're used to how the Plots pane works, and having given several demos recently its not a great look to have a big red warning pop up when everything is working as expected.

> Add more text at the end of the message saying something like:
>
> > ([click here](https://github.com/spyder-ide/spyder/issues/21279) if you don't want to see this message anymore).

I would suggest something like

> [Click here](https://spyder-ide.org) to not show this message again

And its a good idea to make the Plots pane a link too (though it would also be nice to have an option to automatically show it whenever plots are generated, and maybe enable that by default to remove the need for the warning entirely).
